src/test/java/io/vertx/test/core/DNSTest.java

```diff
@@ -63,6 +66,27 @@ public void testIllegalArguments() throws Exception {
     dnsServer.stop();
   }
 
+  @Test
+  public void testDefaultDnsServer() throws Exception {
```

Comment:
why isn't the test using createDnsClient() ?

Reply:
In the test, I have tried to test that the first DNS server in a list is our fake DNS server. 

I m agree with you. We should test it with direct _createDnsClient()_ call.

I have just made some changes. Is that correct?